templates/advantage/table/_manage-monthly.html

```diff
@@ -45,13 +45,15 @@ <h5 class="u-no-padding--top" {% if not mobile %}style="margin-bottom: .5rem"{%
     {% if not mobile %}
       <hr />
     {% endif %}
+    <!-- It conflicts with the total contract expiry date, if yearly purchase exists
```

Comment:
I thought we were only hiding it when its a month contract not entirely?

Reply:
The file I am changing is: `templates/advantage/table/_manage-monthly.html`

Everything I change in that file affects only monthly subscriptions. However yearly subscriptions do not have this notification, so technically I am hiding it for all subscriptions. 

Once the /advantage redesign kicks in, we can reconsider adding this back.

Reply:
Do we want to just remove the code instead of comment it out? We can always look in git history if we need to reintroduce it

Reply:
@anthonydillon show I remove it? I had a feeling that if we remove it we would forget about it forever. So I chose to just comment it out.

Reply:
> The file I am changing is: templates/advantage/table/_manage-monthly.html

Ah thanks, makes sense.

> @anthonydillon show I remove it? I had a feeling that if we remove it we would forget about it forever. So I chose to just comment it out.

I would prefer to remove it as random commented out code quickly becomes confusing and is rarely useful when reimplementing.